keyboards/lazydesigners/dimple/dimple.c

```diff
@@ -22,3 +22,8 @@ void dimple_led_on() {
 void dimple_led_off() {
   DDRE &= ~(1 << 6); PORTE &= ~(1 << 6);
```

Comment:
```suggestion
  writePinLow(E6);
```

And the "on" function should be:
```c
  writePinHigh(E6);
```